<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>536</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EasyRepSift - Routing</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources_routing.qrc">
    <normaloff>:/icons/resources/app_logo.png</normaloff>:/icons/resources/app_logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Dialog background */
QDialog {
    background-color: #a595fa;
}

/* GroupBox: rounded, white font */
QGroupBox#groupBox {
    border: 2px solid #7c6fc2;
    border-radius: 20px;
    margin-top: 20px;
    padding: 14px 10px 10px 10px;
    font-family: 'Segoe UI', Segoe, Arial, sans-serif;
    font-size: 14pt;
    color: #fff;
}

/* GroupBox title: left, no background, white */
QGroupBox#groupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top left;
    left: 15px;
    padding: 0 5px;
    color: #fff;
    font-weight: bold;
    font-family: 'Segoe UI', Segoe, Arial, sans-serif;
    font-size: 12pt;
}

/* Input field for routingID: rounded, slightly darker background, white text */
QLineEdit#routingID_input {
    font-family: 'Segoe UI', Segoe, Arial, sans-serif;
    font-size: 12pt;
    color: #fff;
    background-color: #7c6fc2;
    border-radius: 10px;
    padding: 0px 8px;
    border: 2px solid #7c6fc2;
}

/* Info label inside groupBox: white, italic, slightly smaller */
QLabel#example_info {
    color: #fff;
    font-size: 10pt;
    font-style: italic;
    font-family: 'Segoe UI', Segoe, Arial, sans-serif;
}

/* OK Button: prominent, white text, hover effect */
QPushButton#ok_button {
    font-family: 'Segoe UI', Segoe, Arial, sans-serif;
    font-size: 15pt;
    color: #fff;
    background-color: #7c6fc2;
    border-radius: 10px;
    padding: 6px 18px;
    font-weight: bold;
    border: 2px solid transparent;
    transition: border 0.2s, box-shadow 0.2s;
}
QPushButton#ok_button:hover {
    background-color: #9587e3;
    border: 2px solid #fff;
    box-shadow: 0 0 0 2px #fff;
}
QPushButton#ok_button:pressed {
    background-color: #6a5fb2;
}

/* CANCEL Button: prominent, white text, hover effect */
QPushButton#cancel_button {
    font-family: 'Segoe UI', Segoe, Arial, sans-serif;
    font-size: 15pt;
    color: #fff;
    background-color: #7c6fc2;
    border-radius: 10px;
    padding: 6px 18px;
    font-weight: bold;
    border: 2px solid transparent;
    transition: border 0.2s, box-shadow 0.2s;
}
QPushButton#cancel_button:hover {
    background-color: #9587e3;
    border: 2px solid #fff;
    box-shadow: 0 0 0 2px #fff;
}
QPushButton#cancel_button:pressed {
    background-color: #6a5fb2;
}

/* Header/Footer labels: white, bold, larger */
QLabel#logo_header, QLabel#logo_footer {
    color: #fff;
    font-size: 18px;
    font-weight: bold;
}

/* Frames: transparent */
QFrame {
    background: transparent;
    border: none;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame1_header">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="logo_header">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources_routing.qrc">:/logos/resources/header_logo.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame2_body">
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Introduce el conector a filtrar:</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLineEdit" name="routingID_input">
           <property name="statusTip">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="example_info">
           <property name="font">
            <font>
             <family>Segoe UI</family>
             <pointsize>10</pointsize>
             <italic>true</italic>
            </font>
           </property>
           <property name="text">
            <string>Para m√°s de un conector, separar por comas (p. ej. 61014,61038).</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignmentFlag::AlignRight">
       <widget class="QFrame" name="frame_okcancel">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="ok_button">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources_routing.qrc">
             <normaloff>:/icons/resources/ok.png</normaloff>:/icons/resources/ok.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_button">
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../resources_routing.qrc">
             <normaloff>:/icons/resources/close.png</normaloff>:/icons/resources/close.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame3_footer">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="logo_footer">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>100</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resources_routing.qrc">:/logos/resources/footer_logo.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources_routing.qrc"/>
 </resources>
 <connections/>
</ui>
